### ISSUE TITLE
Installing Jupyter into new Python install on Windows 10 VM failes silently

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#12703]
https://github.com/microsoft/vscode-jupyter/issues/11854
I tried this on a VM and the installation of Jupyter packages fails, we should detect this and notify the user

**Comment by DonJayamanne** [comment]
Error in terminal when trying this manually
```
Installing collected packages: jedi, comm, cffi, bleach, beautifulsoup4, asttokens, anyio, stack-data, nbformat, jupyter-server-terminals, jupyter-client, arrow, argon2-cffi-bindings, nbclient, isoduration, ipython, argon2-cffi, nbconvert, ipykernel, qtconsole, jupyter-events, jupyter-console, ipywidgets, jupyter-server, notebook-shim, nbclassic, notebook, jupyter
ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: 'C:\\Users\\donjayamanne\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python310\\site-packages\\jedi\\third_party\\typeshed\\third_party\\2and3\\requests\\packages\\urllib3\\packages\\ssl_match_hostname\\_implementation.pyi'
HINT: This error might have occurred since this system does not have Windows Long Path support enabled. You can find information on how to enable this at https://pip.pypa.io/warnings/enable-long-paths


[notice] A new release of pip available: 22.3.1 -> 23.0
[notice] To update, run: C:\Users\donjayamanne\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\python.exe -m pip install --upgrade pip

**Comment by DonJayamanne** [comment]
[ms-toolsai-jupyter-insiders.vsix.zip](https://github.com/microsoft/vscode-jupyter/files/10726960/ms-toolsai-jupyter-insiders.vsix.zip)
